Src/AutoFixture/RandomNumericSequenceGenerator.cs

```diff
@@ -53,7 +51,17 @@ public RandomNumericSequenceGenerator(params long[] limits)
 
             if (limits.Length < 2)
             {
-                throw new ArgumentException("The limit must be a sequence of two or more integers.");
+                throw new ArgumentException("limits must be two or more Int64.", "limits");

```

Comment:
I have to admit I don't like the term "integer" because it implies 'int', which is not the same as a 'long', so I generally use the BCL type names outside code; but I can change it back to integer or maybe simply "number"?

Will do for the capital L.